guides/v2.2/magento-functional-testing-framework/release-2/test.md

```diff
@@ -5,24 +5,26 @@ version: 2.2
 github_link: magento-functional-testing-framework/release-2/test.md
 functional_areas:
  - Testing
-mftf-release: 2.2.0
+mftf-release: 2.3.0
 ---
 
 _This topic was updated due to the {{page.mftf-release}} MFTF release._
 {: style="text-align: right"}
 
-Test cases in the Magento Functional Testing Framework (MFTF) are defined in XML as [`<tests>`](#test-tag).
-`<tests>` is a [Codeception test container](https://codeception.com/docs/07-AdvancedUsage) that contains multiple individual tests with test metadata and before and after actions.
+Test cases in the Magento Functional Testing Framework (MFTF) are defined in XML as [`<tests>`].
+`<tests>` is a [Codeception test container][Codeception] that contains multiple individual tests with test metadata and before and after actions.
 
 MFTF `<tests>` is considered a sequence of actions with associated parameters.
-Any failed [assertion](./test/assertions.html) within a test constitutes a failed test.
+Any failed [assertion] within a test constitutes a failed test.
 
-<div class="bs-callout bs-callout-info" id="info" markdown="1">
-`<before>` and `<after>` hooks are not global within `<tests>` like in MFTF 1.0.
+{%
+include note.html
+type = "info"
+content='`<before>` and `<after>` hooks are not global within `<tests>` like in MFTF 1.0.
```

Comment:
`<tests>` ~~like~~
`<tests>`, which was the case

Reply:
removed "like in MFTF 1.0"